pagopa/io-app-design-system
# fix: PictogramFocusOn typo fix

## Short description
fixed typo in said component

## List of changes proposed in this pull request
- typo fix
## How to test
use PictogramFocusOn in a component and make sure the correct pictogram is rendered
